portfolio/src/test/java/com/google/sps/HashTableTest.java

```diff
@@ -61,13 +61,13 @@ public void multipleBlogPostsForFewTags() {
     BlogHashMap actual = new BlogHashMap();
     List<BlogMessage> temp = new ArrayList<>();
     for(int i = 0; i < 4; i++){
```

Comment:
Nit: This is me nitpicking and I do not have strong feeling (especially in testing code), so your call.

I see that the constant 4 is used four times. It would be good to use a local variable like:

int iterations = 4;

And then use the variable.